docs/articles/lwc/localdev.md

```diff
@@ -0,0 +1,232 @@
+
+```sh
+sfdx force:org:create -s -f config/project-scratch-def.json -a "LWC"
+```
+
+“LWC” is an alias for the scratch org that you can use in other Salesforce CLI commands.
+
+To create a scratch org, specify a scratch org definition file. This example uses the scratch org definition file, project-scratch-def.json that is included in `lwc-recipes`. For other projects, create your own. For more information, see the instructions for [Create Scratch Orgs](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_create.htm) in the *Salesforce DX Developer Guide*.
+
+9. Start the server.
+
+```sh
+sfdx force:lightning:lwc:start
+```
+
+For more information on this command, view Help for the Local Development Server by running `sfdx force:lightning:lwc:start --help`.
+
+10. View the server at [http://localhost:3333/](http://localhost:3333/). 
+
+## Troubleshooting
+
+```sh-session
+$ sfdx force:lightning:lwc:start
+Starting LWC Local Development.
+    Dev Hub Org: mydevhub
+    Scratch Org: undefined - We can't find an active scratch org for this Dev Hub. Create one by following the steps in Create Scratch Orgs in the Salesforce DX Developer Guide (https://sfdc.co/cuuVX4) or the Local Development Server Getting Started.
+```
+
+If you see this error, make sure that you authenticate to your Dev Hub and create a scratch org. 
+
+## Configuration for Projects (Optional)
+
+SFDX automatically configures your project out of the box, but if you want to provide additional information for the server, add a localdevserver.config.json file at the base of your project. 
+
+Here's an example that shows the available configuration parameters.
+
+```json5
+{
+    // What namespace to use referencing your Lightning Web Components
+    "namespace": "c",
+
+    // Name of the component to load in the default container
+    "main": "app", 
+
+    // Where are your component files. If you have a namespace, specify the directory the namespace folder is in.
+    "modulesSourceDirectory": "src/", 
+
+    // Where are your static assets.
+    "staticResourcesDirectory": "",
+
+    // The address port for your local server. Defaults to 3333
+    "port": 3333,
+
+    // Optional path to the custom labels file
+    "customLabelsFile": "labels/CustomLabels.labels-meta.xml",
+}
+```
+
+**Working With Salesforce Modules**
+-----------------------------------
+
+## Supported Modules
+
+| **Module Name**      |    | **Local Development Behavior** |
+| ----------- |     | ----------- |
+| `@salesforce/resourceUrl`      |  | Imports static resources into your Salesforce org using the structure: `import <resourceName> from '@salesforce/resourceUrl'`. Static resources are copied and served from the SFDX project location on your filesystem to the local development server.       |
```

Comment:
Can we clarify, "Imports static resources into your Salesforce org using the structure"? Local development does not import static resources into the org.